controllers/ibmvpccluster_controller.go

```diff
@@ -73,9 +72,8 @@ func (r *IBMVPCClusterReconciler) Reconcile(ctx context.Context, req ctrl.Reques
 		return ctrl.Result{}, nil
 	}
 
-	// Create the scope.
-	// TODO: Will be removed once we find a better way of overriding the service endpoint, generate via spec.
-	svcEndpoint := os.Getenv("SERVICE_ENDPOINT")
```

Comment:
there are other occurrences of this `SERVICE_ENDPOINT` in the docs, lets remove them as well.

Reply:
Done.